modules/ROOT/pages/_partials/link-anypoint-platform-accounts.adoc

```diff
@@ -0,0 +1,110 @@
+// tag::profileLinkingContextForUsers[]
+[[profile-linking-context-users]]
+= Linking Your Anypoint Platform Profiles
+
+When you log in to Anypoint Platform for the first time using an external identity provider (IdP), Anypoint Platform locates potential user accounts matched by your email address that is associated with the IdP profile. You are prompted to link your IdP profile with the existing accounts. Your options for linking profiles vary based on whether the profiles use the Anypoint identity provider (IdP) or an external IdP that uses single sign-on (SSO). Anypoint Platform supports:
+
+* Linking an SSO login to a local profile by using credentials (login managed by Anypoint IdP)
+* Linking an SSO login to another SSO login that uses a different IdP to log in
+* Linking an SSO login to multiple other profiles that each use different IdPs to log in
+
+Anypoint Platform does not support linking multiple profiles that use the same IdP. The IdP must be an OpenID Connect (OIDC) provider. You can link up to 25 user profiles. This feature is available only if an organization administrator has enabled it.
+
+// end::profileLinkingContextForUsers[]
+
+// tag::profileLinkingForSSOtoLocal[]
+[[sso-to-local]]
+= Linking an SSO Login to a Local Profile
+
+When you attempt to log in using SSO and there is another user profile that shares the same email address but uses Anypoint Platform credentials to log in, you are prompted to link the profiles.
+
+. In the Anypoint Platform login page, click *Sign in with <your identity provider>*.
+. If prompted, enter your credentials for your IdP. +
+If your SSO credentials share an email address with a profile that is managed by the Anypoint IdP, you are prompted to log in. If you are not prompted, your local profile uses a different email address than your SSO login.
++
+. In the *Link your Account* page, enter your credentials for your local Anypoint Platform profile.
+. Click *Link Accounts*.
++
+You can now view your linked profiles in the *Identity Providers* section of your user profile.
+
+// end::profileLinkingForSSOtoLocal[]
+
+// tag::profileLinkingSSOtoSSO[]
+[[sso-to-sso]]
+= Linking an Existing SSO Login to Another SSO Login
```

Comment:
High level comment about the structure of this page. Currently we have 3 sections for different linking scenarios, which is good, but, this may imply that the end user is able to initiate one of the 3 cases or they are able to choose. In fact, the end user doesn't have the ability to choose which cases they would step into. Email matching can be a black box to end users unless they are super clear about their emails and accounts in Anypoint. 

With this in mind, is it possible to unify the 3 cases in to just one workflow, and talk about potential options for an end user. A sample workflow is: (just an example)
1. Do sso login. 
2. You may be prompted with a 'Link an account' for you choose which existing account you want to login as and the selected user account will be linked to the sso login. 
3. After selecting a user, you may be promoted to authenticate using existing credentials linked/attached to this user account 
4. ..

Reply:
Sure. We don't have to include all of them. They're sectioned off, as this page isn't published as-is.